environments/autostart.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Autostart"
+description: Learn how to configure automated environment rebuilds.
+---
+
+Coder automatically [turns off](../admin/environment-management/shutdown.md)
+unused environments to help you save on cloud usage costs. Turning off
+environments, however, requires you to turn them on manually the next day.
+Autostart can help eliminate this manual process.
+
+Autostart also allows you to configure automated environment rebuilds at a time
+that best suits your workflow. You can now expect your environments to be ready
+for you at the start of each workday.
+
+## Criteria for Autostart
+
+Your environment must meet the following criteria for you to use autostart:
```

Comment:
IMO, I think we should always refer to this feature with title case. @khorne3 thoughts?

```suggestion
Your environment must meet the following criteria for you to use Autostart:
```

Reply:
I think that's fine